apps/src/music/views/MusicView.jsx

```diff
@@ -324,6 +327,13 @@ class UnconnectedMusicView extends React.Component {
     );
   };
 
+  onVideoClosed = () => {
+    this.setState({showingVideo: false});
```

Comment:
Is there any way to reopen the video once closed (and do we want that)?

Reply:
My understanding was that this was a one-off for now, but @moneppo let us know if we want to be able to reopen it.